cmdtests/common_test.go

```diff
@@ -0,0 +1,112 @@
+// +build integration
+
+package cmdtests_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/src-d/engine/cmdtests"
+	"github.com/stretchr/testify/suite"
+)
+
+type StreamLinifierTestSuite struct {
+	suite.Suite
+}
+
+func TestStreamLinifierTestSuite(t *testing.T) {
+	suite.Run(t, &StreamLinifierTestSuite{})
+}
+
+func (s *StreamLinifierTestSuite) TestOneLineOnOneMessage() {
+	require := s.Require()
+
+	sl := cmdtests.NewStreamLinifier(500 * time.Millisecond)
+
+	in := make(chan string)
+	messagesNum := 3
+
+	go func() {
+		for i := 1; i <= messagesNum; i++ {
+			in <- fmt.Sprintf("line %d\n", i)
+		}
+
+		close(in)
+	}()
+
+	out := sl.Linify(in)
+	counter := 1
+	for m := range out {
+		require.Equal(m, fmt.Sprintf("line %d", counter))
+		counter++
+	}
```

Comment:
looks like you missed the check for number on produced lines (which must be equal to `messagesNum`)

Reply:
Thanks! Addressed [here](https://github.com/src-d/engine/pull/432/commits/3ce3ebb3d844900e3233f25efac76aa7600ca268).